src/util/file-filter.js

```diff
@@ -0,0 +1,87 @@
+/* @flow */
```

Comment:
I'm not sure how should we fix it. Just explicitly ignore all content inside the folder?
```diff
    filesToIgnore = [
      '**/*.xpi',
      '**/*.zip',
      '**/.*', // any hidden file
+     '**/.*/**/*', // any content in the hidden folder
      '**/node_modules',
+     '**/node_modules/**/*',
    ],
```